xen-hvm-stubdom-legacy.spec.in

```diff
@@ -0,0 +1,291 @@
+Patch904: patch-stubdom-linux-v3-0005-libxl-Handle-Linux-stubdomain-specific-QEMU-options.patch
+Patch905: patch-stubdom-linux-v3-0006-libxl-write-qemu-arguments-into-separate-xenstore-keys.patch
+Patch906: patch-stubdom-linux-v3-0007-libxl-create-vkb-device-only-for-guests-with-graphics-output.patch
+Patch907: patch-stubdom-linux-v3-0008-xl-add-stubdomain-related-options-to-xl-config-parser.patch
+Patch908: patch-stubdom-linux-v3-0009-tools-libvchan-notify-server-when-client-is-connected.patch
+Patch909: patch-stubdom-linux-v3-0010-libxl-typo-fix-in-comment.patch
+Patch910: patch-stubdom-linux-v3-0011-libxl-move-xswait-declaration-up-in-libxl_internal.h.patch
+Patch911: patch-stubdom-linux-v3-0012-libxl-use-vchan-for-QMP-access-with-Linux-stubdomain-libxl__ev_qmp_-version.patch
+Patch912: patch-stubdom-linux-v3-0013-libxl-use-vchan-for-QMP-access-with-Linux-stubdomain-non-async-version.patch
+Patch913: patch-stubdom-linux-v3-0014-libxl-add-save-restore-support-for-qemu-xen-in-stubdomain.patch
+Patch914: patch-stubdom-linux-v3-0015-tools-add-missing-libxenvchan-cflags.patch
+Patch915: patch-stubdom-linux-v3-0016-libxl-add-locking-for-libvchan-QMP-connection.patch
+# modified vs upstream
+Patch916: patch-stubdom-linux-v3-0017-libxl-require-qemu-in-dom0-even-if-stubdomain-is-in-use.patch
+
+# MSI fixes
+Patch920: patch-stubdom-msi-0001-libxl-do-not-attach-xen-pciback-to-HVM-domain-if-stu.patch
+Patch921: patch-stubdom-msi-0002-libxl-attach-PCI-device-to-qemu-only-after-setting-p.patch
+Patch922: patch-stubdom-msi-0003-libxl-don-t-try-to-manipulate-json-config-for-stubdo.patch
+Patch923: patch-stubdom-msi-0004-xen-x86-Allow-stubdom-access-to-irq-created-for-msi.patch
+Patch924: patch-stubdom-msi-0005-xen-x86-add-PHYSDEVOP_msi_set_enable.patch
+Patch925: patch-stubdom-msi-0006-tools-libxc-add-wrapper-for-PHYSDEVOP_msi_set_enable.patch
+
+# Qubes specific patches
+Patch1001: patch-stubdom-vbd-non-dom0-backend.patch
+Patch1002: patch-xen-no-downloads.patch
+Patch1003: patch-xen-hotplug-external-store.patch
+Patch1004: patch-xen-stubdom-qubes-gui.patch
+Patch1005: patch-stubdom-lwip-fix-for-dhcp.patch
+Patch1006: patch-xen-libxl-qubes-minimal-stubdom.patch
+Patch1007: patch-xen-disable-dom0-qemu.patch
+Patch1009: patch-xenconsoled-enable-logging.patch
+#TODO Patch1110: patch-stubdom-linux-libxl-suspend.patch
+Patch1011: patch-xen-hotplug-qubesdb-update.patch
+Patch1012: patch-tools-hotplug-drop-perl-usage-in-locking-mechanism.patch
+Patch1013: patch-stubdom-linux-libxl-use-EHCI-for-providing-tablet-USB-device.patch
+Patch1014: patch-allow-kernelopts-stubdom.patch
+Patch1015: patch-libxl-readonly-disk-scsi.patch
+Patch1016: patch-tools-xenconsole-replace-ESC-char-on-xenconsole-outp.patch
+
+Patch1020: patch-stubdom-linux-config-qubes-gui.patch
+Patch1021: patch-stubdom-linux-libxl-do-not-force-qdisk-backend-for-cdrom.patch
+Patch1022: patch-xen-acpi-slic-support.patch
+
+Patch1100: patch-fix-errors-on-ambiguous-python-shebang.patch
```

Comment:
Is rpm complaining without this? The package shouldn't contain any python files...